DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -47,9 +47,35 @@ namespace
 		return TCHAR_TO_UTF8(*String);
 	}
 
+	EDolbyIOParticipantStatus ToEDolbyIOParticipantStatus(std::optional<dolbyio::comms::participant_status> Status)
```

Comment:
Why convert the optional? This optional will never be empty in the current logic flow since you are always checking beforehand.

Reply:
For safety and it is checked before call to the caller of it so relying on that check is error-prone to changes by unconscious this dependency.